src/content/docs/identityserver/upgrades/identityserver4-to-duende-identityserver-v7.mdx

```diff
@@ -1,8 +1,8 @@
 ---
 title: "IdentityServer4 to Duende IdentityServer v7.3"
```

Comment:
The label and the title still seemed mismatched. Are you sure this is right?

Reply:
You are right